src/generators/cfml.ts

```diff
@@ -0,0 +1,113 @@
+import * as util from "../util.js";
+import type { Request } from "../util.js";
+
+import jsesc from "jsesc";
+
+const quote = (str: string): string => {
+  return jsesc(str, { quotes: "single" }).replace(/"/g, '""');
+};
+
+export const _toCFML = (request: Request): string => {
+  let cfmlCode = "";
+
+  cfmlCode += "httpService = new http();\n";
+  cfmlCode += 'httpService.setUrl("' + quote(request.url as string)  + '");\n';
+  cfmlCode += 'httpService.setCharset("utf-8");\n';
```

Comment:
What is this and what's the reason for adding this line in general? Is this the `charset` part of the `Content-Type: text/html; charset=UTF-8` header? If so, we shouldn't add this line if `hasHeader(request, 'content-type')`. Ideally we could parse the `content-type` header and remove the `charset=UTF-8` and add this line instead if it exists, or just don't add it at all for any reason.

Reply:
I'll just leave it out